h2o-r/h2o-package/R/glm.R

```diff
@@ -129,7 +129,7 @@
 #' # Run GLM of CAPSULE ~ AGE + RACE + PSA + DCAPS
 #' prostate_path = system.file("extdata", "prostate.csv", package = "h2o")
 #' prostate = h2o.importFile(path = prostate_path)
-#' h2o.glm(y = "CAPSULE", x = c("AGE","RACE","PSA","DCAPS"), training_frame = prostate,
```

Comment:
This is the wrong file to update. The changes to this and other algos have to be made in:
/h2o-3/h2o-bindings/bin/custom/R